lib/features/children/generosity_challenge/cubit/generosity_challenge_cubit.dart

```diff
@@ -28,6 +48,7 @@ class GenerosityChallengeCubit extends Cubit<GenerosityChallengeState> {
       );
 
   void dayDetails(int dayIndex) => emit(
+        //maybe here i need logic
```

Comment:
I'd prefer if you'd elaborate the comment or if you make a ticket in Linear and then add a TODO, like:

// TODO KIDS-999 title

Reply:
that is not mine, but I'll delete it

Reply:
deleted